From 1cf0030efbf067ca4960a443211a91c586992706 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 30 Aug 2018 12:46:18 -0700
Subject: [PATCH] TRITON-741 Node.js 'querystring.parse' considered surprising
 in node v6 (drop unused querystring imports) Reviewed by: Orlando Vazquez
 <orlando@joyent.com> Approved by: Orlando Vazquez <orlando@joyent.com>

---
 lib/changefeed-filter.js   | 1 -
 lib/cn-filter.js           | 1 -
 lib/net-pool-filter.js     | 1 -
 lib/network-info-filter.js | 1 -
 lib/reaper-stream.js       | 1 -
 lib/ufds-filter.js         | 1 -
 lib/ufds-pool.js           | 1 -
 lib/ufds-watcher.js        | 1 -
 8 files changed, 8 deletions(-)

diff --git a/lib/changefeed-filter.js b/lib/changefeed-filter.js
index 8c576ac..350189a 100644
--- a/lib/changefeed-filter.js
+++ b/lib/changefeed-filter.js
@@ -14,7 +14,6 @@ var assert = require('assert-plus');
 var utils = require('./utils');
 var bunyan = require('bunyan');
 var restify = require('restify-clients');
-var qs = require('querystring');
 
 var consts = require('./consts');
 
diff --git a/lib/cn-filter.js b/lib/cn-filter.js
index b24671a..8b19829 100644
--- a/lib/cn-filter.js
+++ b/lib/cn-filter.js
@@ -15,7 +15,6 @@ var utils = require('./utils');
 var bunyan = require('bunyan');
 var LRUCache = require('lru-cache');
 var restify = require('restify-clients');
-var qs = require('querystring');
 var crypto = require('crypto');
 
 var consts = require('./consts');
diff --git a/lib/net-pool-filter.js b/lib/net-pool-filter.js
index a5c3166..5ecab50 100644
--- a/lib/net-pool-filter.js
+++ b/lib/net-pool-filter.js
@@ -15,7 +15,6 @@ var utils = require('./utils');
 var bunyan = require('bunyan');
 var LRUCache = require('lru-cache');
 var restify = require('restify-clients');
-var qs = require('querystring');
 
 var consts = require('./consts');
 
diff --git a/lib/network-info-filter.js b/lib/network-info-filter.js
index ba2aba4..c3aed21 100644
--- a/lib/network-info-filter.js
+++ b/lib/network-info-filter.js
@@ -15,7 +15,6 @@ var utils = require('./utils');
 var bunyan = require('bunyan');
 var LRUCache = require('lru-cache');
 var restify = require('restify-clients');
-var qs = require('querystring');
 var vasync = require('vasync');
 
 var consts = require('./consts');
diff --git a/lib/reaper-stream.js b/lib/reaper-stream.js
index e87c25f..8cf49ee 100644
--- a/lib/reaper-stream.js
+++ b/lib/reaper-stream.js
@@ -14,7 +14,6 @@ var assert = require('assert-plus');
 var utils = require('./utils');
 var bunyan = require('bunyan');
 var restify = require('restify-clients');
-var qs = require('querystring');
 var EventEmitter = require('events').EventEmitter;
 
 var consts = require('./consts');
diff --git a/lib/ufds-filter.js b/lib/ufds-filter.js
index 51b8f66..0ffa2da 100644
--- a/lib/ufds-filter.js
+++ b/lib/ufds-filter.js
@@ -16,7 +16,6 @@ var bunyan = require('bunyan');
 var LRUCache = require('lru-cache');
 var restify = require('restify-clients');
 var restify_errors = require('restify-errors');
-var querystring = require('querystring');
 var ldap = require('ldapjs');
 
 var ResourceNotFoundError = restify_errors.ResourceNotFoundError;
diff --git a/lib/ufds-pool.js b/lib/ufds-pool.js
index af71050..fc73a71 100644
--- a/lib/ufds-pool.js
+++ b/lib/ufds-pool.js
@@ -15,7 +15,6 @@ var utils = require('./utils');
 var bunyan = require('bunyan');
 var LRUCache = require('lru-cache');
 var restify = require('restify-clients');
-var qs = require('querystring');
 var vasync = require('vasync');
 var EventEmitter = require('events').EventEmitter;
 
diff --git a/lib/ufds-watcher.js b/lib/ufds-watcher.js
index dcbc978..2201968 100644
--- a/lib/ufds-watcher.js
+++ b/lib/ufds-watcher.js
@@ -15,7 +15,6 @@ var utils = require('./utils');
 var bunyan = require('bunyan');
 var LRUCache = require('lru-cache');
 var restify = require('restify-clients');
-var qs = require('querystring');
 var vasync = require('vasync');
 var EventEmitter = require('events').EventEmitter;
 
-- 
2.21.0

